west.yml

```diff
@@ -131,7 +131,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: ac671e8bcd994d2025aa3f4763c0742ca8dadc3e
+      revision: pull/979/head
```

Comment:
Please update SHA and rebase (`b063f193a08d9e94ae7d09aaec065cd96673003e`)

Reply:
done